### PR TITLE
update-deliveroo-overview.md

### PR DESCRIPTION
**Pull request by janainawittner**
It is important to refer to "convenience stores" as Deliveroo reviewed the description we made of them on the Apps page and insisted on the change. 
I also think we should include the Integration Feature section here.
Prerequisites section go last. I think we should keep this section in the Overview page, and not remove it as we had discussed with Antoine at some point.

### CODE REVIEW

File: content/apps/deliveroo/en/overview.md [hunk 1]
```diff
@@ -3,31 +3,43 @@ position: 1
...
+  description: Deliveroo overview and why restaurants and convenience stores should connect it to HubRise. With a connection orders are sent to your working tools - EPOS, KDS.
 path_override: /
 app_info:
-  category: Online ordering and food delivery
-  availability:
-  price_range:
+  category: Food Ordering Platforms
+  availability: Worldwide
+  price_range: Free for HubRise subscribed users
   website: https://deliveroo.co.uk
```
Comment (janainawittner) [inline content/apps/deliveroo/en/overview.md:13]: I suggest to make the category correspond to what we have in the Apps page. I think it this is important for consistency. 

Why not include availability and Price too?

File: content/apps/deliveroo/en/overview.md [hunk 2]
```diff
@@ -3,31 +3,43 @@ position: 1
 title: Overview
```
Comment (janainawittner) [inline content/apps/deliveroo/en/overview.md:3]: Page title should be changed from Deliveroo to Deliveroo Bridge. I mean, the title above the menu in the right handside of the Deliveroo documentation.

Comment (antmonnier) [reply content/apps/deliveroo/en/overview.md:3]: (Agreed. It should be done in the same way as PixelPoint Bridge.)

File: content/apps/deliveroo/en/overview.md [hunk 3]
```diff
@@ -3,31 +3,43 @@ position: 1
...
 layout: documentation
 meta:
-  title: Deliveroo Connection to HubRise Overview
-  description: Overview of Deliveroo and how to connect it to HubRise.
+  title: Deliveroo Connection to HubRise - Overview
+  description: Deliveroo overview and why restaurants and convenience stores should connect it to HubRise. With a connection orders are sent to your working tools - EPOS, KDS.
 path_override: /
```
Comment (janainawittner) [inline content/apps/deliveroo/en/overview.md:8]: There should be a gallery: section above this to include screenshot, same as I have done on this page: https://www.hubrise.com/apps/zelty

Please include two screenshots from Deliveroo. Could be one of the restaurants conneting: https://www.ubereats.com/ch/geneva/food-delivery/moshimoshi-carouge/CXJ6X53JQkqpZxAo9033EA

And one of the Deliveroo back office. 
@atorin, if you have no access, I think Antoine has one.

File: content/apps/deliveroo/en/overview.md (hunk 2 above)
Comment (janainawittner) [reply content/apps/deliveroo/en/overview.md:3]: @atorin, regarding the diagram, you can find the source in the Drive https://drive.google.com/drive/u/1/folders/1YR8rkatzK-9r6bowt9_yy0ml06FC0PLM

Comment (atorin) [reply content/apps/deliveroo/en/overview.md:3]: @janainawittner OK, I agree with the change to the title. And also the diagram seems a great idea, we should add it for all the integrations.

File: content/apps/deliveroo/en/overview.md (hunk 1 above)
Comment (atorin) [reply content/apps/deliveroo/en/overview.md:13]: I didn't think of the category in the Apps page, it makes perfect sense!

OK for pricing, but are you sure that the availability is really "Worldwide"?

File: content/apps/deliveroo/en/overview.md (hunk 3 above)
Comment (atorin) [reply content/apps/deliveroo/en/overview.md:8]: But do we want a screenshot of Deliveroo or Deliveroo Bridge?

File: content/apps/deliveroo/en/overview.md [hunk 4]
```diff
@@ -3,31 +3,43 @@ position: 1
...
 app_info:
-  category: Online ordering and food delivery
-  availability:
-  price_range:
+  category: Food Ordering Platforms
+  availability: Worldwide
+  price_range: Free for HubRise subscribed users
   website: https://deliveroo.co.uk
-  contact:
+  contact: contact@hubrise.com
```
Comment (atorin) [inline content/apps/deliveroo/en/overview.md:14]: support@hubrise.com?

File: content/apps/deliveroo/en/overview.md [hunk 5]
```diff
@@ -3,31 +3,43 @@ position: 1
...
 ---
 
 ## Description
 
-Deliveroo is an online ordering and food delivery platform based in the United Kingdom but operating in 13 different countries in Europe, Asia, and Oceania.
+Deliveroo is an online delivery platform for restaurants and convenience stores. They are based in the United Kingdom but operating in 13 different countries in Europe, Asia, and Oceania.
```
Comment (atorin) [inline content/apps/deliveroo/en/overview.md:19]: It is based in the United Kingdom but operates...

File: content/apps/deliveroo/en/overview.md [hunk 6]
```diff
@@ -3,31 +3,43 @@ position: 1
...
 
-Restaurants can benefit from Deliveroo's large pool of customers to grow their business and find new clients.
-At the same time, they can concentrate on what they do best (preparing food) while letting Deliveroo take care of the logistics of food delivery.
+Restaurants and convenience stores can benefit from Deliveroo's large pool of customers to grow their business and find new clients.
+At the same time, they can concentrate on what they do best (preparing food, selling groceries) while Deliveroo take care of the online ordering solution as associated marketing as well as the delivery logistics.
```
Comment (atorin) [inline content/apps/deliveroo/en/overview.md:22]: - while Deliveroo takes care
- online ordering solution and associated marketing

File: content/apps/deliveroo/en/overview.md [hunk 7]
```diff
@@ -3,31 +3,43 @@ position: 1
...
+IMPORTANT NOTE: Product catalog and product availability available in HubRise cannot currently be pushed into Deliveroo, but HubRise is working on it.
+
+--------
 
 ## Why Connect?
 
-Connecting your Deliveroo online restaurant to HubRise allows you to receive Deliveroo orders directly to your POS connected with HubRise.
-You will be able to manage Deliveroo orders seamlessly from your existing setup, as you would do with all the other orders.
+Connecting your Deliveroo online restaurant or convenience store to HubRise allows you to receive Deliveroo orders directly to your EPOS or Kitchen Display Screen also connected with HubRise.
+You will be able to manage Deliveroo orders alongside other orders, using your own tools. Deliveroo orders would flow through your existing setup, as you would do with any other order.
```
Comment (atorin) [inline content/apps/deliveroo/en/overview.md:39]: Since we are using "will", I would rephrase the last sentence as:
"Deliveroo orders will flow through your existing setup, as all other orders already do."
(or something similar).

File: content/apps/deliveroo/en/overview.md (hunk 2 above)
Comment (atorin) [reply content/apps/deliveroo/en/overview.md:3]: Here's the new diagram: https://docs.google.com/drawings/d/19t0gSjjLk2TW42RARSSiO5Op6nvYzE51VQ6cFFs8kYU
Feel free to make amendments.

**Comment by antmonnier** [comment]
@atorin @janainawittner Here are my takes re the various points discussed above:

- I'm having second thoughts on changing this title to "Deliveroo Bridge"
image. I think the current title makes more sense to the user.

- Alberto, your diagram is fine, you can include it, along with a quick description of the "Deliveroo Bridge" as mentioned in a note above

- Screenshots: I think we should include 2 screenshots from the Deliveroo website and 1 screenshot of our bridge.

- This section at the bottom should refer to the main solution (here Deliveroo, not the bridge). The availability zone (13 countries) should be moved from the app description to this section. Price range should be Deliveroo pricing (not ours), but I think we can remove it since a user reading this page likely already has an idea how much they charge. We can also remove Contact.
image

These are just my hot takes, please let me know what you think...
